pkg/util/slice/slice.go

```diff
@@ -48,3 +48,13 @@ func ShuffleStrings(s []string) []string {
 	}
 	return shuffled
 }
+
+// StringInSlice checks if str is in slice s
+func StringInSlice(str string, s []string) bool {
+	for _, v := range s {
```

Comment:
minor optimization: this copies the string values.  If you use the index I think you save the copies.

```
for i := range s {
  is str == s[i]
```